src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -639,6 +639,72 @@ featurizers:
       - input_type: TInputDouble
         output_type: double
 
+    # ----------------------------------------------------------------------
+  - name: MaxHorizonFeaturizer
+    estimator_name: MaxHorizonFeaturizer

```

Comment:
Make sure that you run `Tools/CodeGenerator/Build.py` on this before created a PR. The changes aren't valid as they are currently implemented.

Reply:
yes